Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -490,9 +491,29 @@ void MenuBar::AddOptionsMenu()
   m_change_font = AddAction(options_menu, tr("&Font..."), this, &MenuBar::ChangeDebugFont);
 }
 
+void MenuBar::InstallUpdateManually()
+{
+  auto& track = SConfig::GetInstance().m_auto_update_track;
+  auto previous_value = track;
+
+  track = "dev";
```

Comment:
Do we really want to force an update to the latest dev version even for people who have selected another track?

Reply:
Well it's probably correct to assume they want to install the latest version if they choose to check for updates. Furthermore most people using this feature will not have a track selected anyway.